lib/thrift/protocols/binary.ex

```diff
@@ -53,10 +49,9 @@ defmodule Thrift.Protocols.Binary do
   def serialize(_, nil) do
     []
   end
-  def serialize(:bool, value) do
-    value = bool_to_int(value)
-    <<value::8-signed>>
-  end
+  def serialize(:bool, nil),   do: <<0::8-signed>>
```

Comment:
Now that I think about it - should this clause ever get called?  If a value is `nil` then it's not set and should be skipped.